pkg/cli/rack.go

```diff
@@ -88,6 +88,19 @@ func init() {
 	})
 }
 
+func validateParams(params map[string]string) error {
+	if params["high_availability"] != "" {
+		return errors.New("the high_availability parameter is only supported during rack installation")
+	}
+
+	srdown, srup := params["ScheduleRackScaleDown"], params["ScheduleRackScaleUp"]
```

Comment:
For v3, the Rack params should use Snake Case rather than Camel Case 
```suggestion
	srdown, srup := params["schedule_rack_scale_down"], params["schedule_rack_scale_up"]
```

Reply:
@beastawakens The ScheduleRackScaleDown and ScheduleRackScaleUp are only going to be used by v2 rack.
This validation is in case someone with v2 rack tries to update through the cli v3.

Reply:
Okay, fair enough.  We should also look to add the other v2 params with similar restrictions (`VPCCIDR` and `Private` spring to mind) at a future point, but no need to complicate this PR with it for now.